### ISSUE TITLE
Color when importing ply 

### ISSUE THREAD

**Issue by Eulmiac** [SBCV/Blender-Addon-Photogrammetry-Importer#11]
Hi,
First of all, thanks a lot for this addon ! It is nice being able to map a particle system onto the point cloud !
Somehow I'm not able to see the colors from the photogrammetry done with metashape...
Is this normal when I'm not doing the point cloud calculation within Blender with this addon ?

**Comment by SBCV** [comment]
Which file format are you using? Could you provide an example?

**Comment by SBCV** [comment]
Furthermore, if there is a suitable file format supported by Metashape we could also think about extending this addon to support that format (so you can import the cameras etc as well). 

Since I got no copy of the software, you would have to provide an example of the [sceaux](https://github.com/openMVG/ImageDataset_SceauxCastle) castle.

**Comment by Eulmiac** [comment]
I exported a ply file.

![04](https://user-images.githubusercontent.com/61442500/75903399-83fae600-5e0f-11ea-9d4e-c74700857e0b.jpg)

In blender :
![05](https://user-images.githubusercontent.com/61442500/75903411-88270380-5e0f-11ea-8e47-eb171319c2f8.jpg)

**Comment by SBCV** [comment]
I think we should first fix that ply issue. 
Can you see the colors of the point cloud, if you import the [example ply file](https://github.com/SBCV/Blender-Addon-Photogrammetry-Importer/blob/master/examples/Example.ply) contained in this repository?

Can you compare the structure of the example ply with the ply file generated by meta shape? 
It should look something like
```
ply
format ascii 1.0
element vertex 7817
property float x
property float y
property float z
property uchar red
property uchar green
property uchar blue
end_header
-0.377804011106491089 -0.52977597713470459 11.3358001708984375 134 149 148
-5.69108009338378906 0.112218998372554779 10.2285003662109375 112 103 86
-1.63103997707366943 -1.88857996463775635 12.339900016784668 71 99 133
-2.82375001907348633 1.21923995018005371 10.2866001129150391 39 60 84
-2.615570068359375 -0.614344000816345215 10.6260004043579102 137 137 138
...
```

Or is it in binary format? 

I need some more information to debug that.

**Comment by Eulmiac** [comment]
thanks for this answer !! I tried with the example PLY and I can see the color !
Indeed, my ply was binary encoded, changing that resolved the issue and I can now see the colors.
Now my problem is the scene itself, it seems that a point cloud with millions of vertices is very slow to manipulate...
It might not be your addon but blender itself, I don't know... Thanks anyway !

**Comment by SBCV** [comment]
Mh, actually the addon is supposed to work with binary ply files as well. Can you share the binary encoded ply file?
The performance is limited by blender. You have to keep in mind, that blender represents each vertex with a real object like a cube. If you want to edit the point cloud you could use [Cloudcompare](https://www.danielgm.net/cc/)

**Comment by Eulmiac** [comment]
Thank you for the suggestion !
I'd be glad to share the file but it's a 70mb file (3M points). Is there a way I can upload it to you ?

**Comment by SBCV** [comment]
If you have drop box you could provide a link for download.
Smaller files can also be attached to the chat - just drag it in. Like so:
[Example.ZIP](https://github.com/SBCV/Blender-Addon-Photogrammetry-Importer/files/4292343/Example.ZIP)

**Comment by Eulmiac** [comment]
Hi, sorry for the late reply ! Here is a link,

https://we.tl/t-c9AH3trpwa

**Comment by SBCV** [comment]
Thanks for sharing, I'll have a look tomorrow.

**Comment by SBCV** [comment]
Two things: 
First: The specs of my computer are too low to move the point cloud / render the all the particles. 
So I subsampled the point cloud with Cloudcompare.  This smaller point cloud I can view / examine with colors in Blender using both binary and txt file formats.
Second: I just noticed that in your Blender screenshot above the 3D view port is set to "Look Dev" and not to "Rendered". This would explain why you have seen no colors. Furthermore, it might be helpful to increase the scale of the particle shape in blender (that determines the size of the point cloud particles). If these are too small, they almost appear black.
